api/test/e2e/global_rule_test.go

```diff
@@ -79,7 +80,7 @@ func TestGlobalRule(t *testing.T) {
 			ExpectStatus:  http.StatusOK,
 			ExpectBody:    "hello world",
 			ExpectHeaders: map[string]string{"X-VERSION": "1.0"},
-			Sleep:         sleepTime,
+			Sleep:         longSleepTime,
```

Comment:
need some comment too.

Reply:
And please update other similar points

Reply:
OK. I had added the comment at base.go already.

https://github.com/starsz/apisix-dashboard/blob/e836046319fe37920f0d65c5a48d19aaa96b7d54/api/test/e2e/base.go#L185-L186